query.go

```diff
@@ -1128,15 +1128,10 @@ func (s *ChainService) sendTransaction(tx *wire.MsgTx, options ...QueryOption) e
 				rejections[*broadcastErr]++
 			}
 		},
-		// Default to 500ms timeout. Default for queryAllPeers is a
+		// Default to 10s timeout. Default for queryAllPeers is a
 		// single try.
-		//
-		// TODO(wilmer): Is this timeout long enough assuming a
-		// worst-case round trip? Also needs to take into account that
-		// the other peer must query its own state to determine whether
-		// it should accept the transaction.
 		append(
-			[]QueryOption{Timeout(time.Millisecond * 500)},
+			[]QueryOption{Timeout(time.Second * 10)},
```

Comment:
Just noting that this will now block for 10 seconds per transaction __rebroadcast__, since peers don't reply to known invs. This should be fine however since [rebroadcasting is done in a separate goroutine](https://github.com/lightninglabs/neutrino/blob/5e09bd9b5d65e90c6ff07aa11b3b9d80d42afb86/pushtx/broadcaster.go#L153).